node/pkg/db/db.go

```diff
@@ -128,6 +128,17 @@ func (d *Database) StoreSignedVAA(v *vaa.VAA) error {
 	return nil
 }
 
+func (d *Database) HasVAA(id VAAID) bool {
```

Comment:
This should return an error if it is anything other than `ErrKeyNotFound`.

Reply:
fixed

Reply:
I don't see a fix.